solutions_builder/template_root/README.md

```diff
@@ -38,8 +38,9 @@ https://console.cloud.google.com/iam-admin/orgpolicies/requireOsLogin?project=$P
 ### Create jump host for the project (Recommended)
 Log in to the jump host
 ```
-export ZONE=us-central1-c #<my-zone>
 sb infra apply 0-jumphost
+export ZONE=$(gcloud compute instances list --format="value(zone)")
```

Comment:
Similar feedback to the previous PR: Please update the env var name to more specific and meaningful. E.g. JUMP_HOST_ZONE